ext/RMagick/extconf.rb

```diff
@@ -186,6 +159,27 @@ def exit_failure(msg)
       exit(1)
     end
 
+    def determine_imagemagick_package
+      exit_failure "Can't install RMagick #{RMAGICK_VERS}. Can't find pkg-config in #{ENV['PATH']}\n" unless find_executable('pkg-config')
```

Comment:
Can we make this a block for clarity:

```rb
unless find_executable('pkg-config')
  exit_failure "..."
end
```

Reply:
I saw this pattern at another place so I just followed the same convention but happy to change it.